ember-headless-form/src/components/-private/types.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
thanks for adding JSDoc to all these! 

JSDoc is soooooo important, cause it shows up in everyone's editors, and super aids in helping folks understand what they're looking at